src/redash_stmo/datasource_health/settings/__init__.py

```diff
@@ -0,0 +1,19 @@
+import os
+import json
+
+def dict_from_string(s):
+    try:
+        return json.loads(s)
+    except ValueError:
+        return {}
+
+# Allow for a map of custom queries to test data source performance and availability.
+# A sample map may look like:
+# {
+#    "1": "select 1;",
+#    "5": "select 1;"
+# }
+CUSTOM_HEALTH_QUERIES = dict_from_string(os.environ.get("REDASH_CUSTOM_HEALTH_QUERIES", ""))
```

Comment:
I'm not sure if it's feasible to store full dicts in strings on the deploy system. Could we iterate over the `os.environ` in the code that uses that and look for a prefix instead? Basically have `REDASH_CUSTOM_HEALTH_QUERY_1`, `REDASH_CUSTOM_HEALTH_QUERY_5`, `REDASH_CUSTOM_HEALTH_QUERY_1234` etc?